views/partials/defaultSettings.ejs

```diff
@@ -0,0 +1,48 @@
+<!-- Apply the settings using the passed EJS user object. -->
```

Comment:
I think I see what's happening here. So the only issue with this approach is that not every page follows the exact same format so some pages (like on aboutus) needed more properties to be modified than others

Reply:
> I think I see what's happening here. So the only issue with this approach is that not every page follows the exact same format so some pages (like on aboutus) needed more properties to be modified than others

Exactly. Specifically, the settings page has a lot of things that need to be done before being sent to the user. Besides just applying the settings to the page, the form needs to have its default values set based on the user's settings selection, which is why the other partial is just used for this page.